pages/index.js

```diff
@@ -34,25 +37,15 @@ class HomePage extends Component {
     const { articles, error } = this.props;
     return (
       <CoreLayout>
-        <p>Articles:</p>
-        <ol>
-          {articles &&
-            articles.map(({ brand, type, locales }) => (
-              <li key={(locales.en && locales.en.slug) || (locales.be && locales.be.slug)}>
-                {type}
-                <br />
-                {locales &&
-                  Object.values(locales).map(({ title, subtitle, text, slug, locale }) => (
-                    <div>
-                      <u>{locale}</u> : <b>{title}</b> : <i>{slug}</i> : {subtitle} : {text} :{' '}
-                      {brand.names[locale]}
-                      <br />
-                    </div>
-                  ))}
-                <br />
-              </li>
-            ))}
-        </ol>
+        <div className="main-page page-container">
+          <div className="page-content">
+            <ArticlesRow articles={articles.slice(0, 4)} />
```

Comment:
same as above, or we can add helper func `getArticleRow(articles)`, for your choice